Library/Formula/sha2.rb

```diff
@@ -0,0 +1,20 @@
+class Sha2 < Formula
+  homepage "http://www.aarongifford.com/computers/sha.html"
+  url "http://www.aarongifford.com/computers/sha2-1.0.1.tgz"
+  sha1 "ffbc69e0b0ab47f70e61aeb14fa16ba2b74dc728"
+
+  option "without-check", "Skip compile-time tests"
+
+  def install
+    system ENV.cc, "-O2", "-DSHA2_UNROLL_TRANSFORM", "-Wall", "-o", "sha2", "sha2prog.c", "sha2.c"
```

Comment:
I've cut the transform. I guess if anyone particularly wants to use the utility on older systems we can put it back, but otherwise I get the feeling you'd rather not include it :smile_cat: